android/src/main/kotlin/com/adyen/checkout/flutter/CheckoutPlatformApi.kt

```diff
@@ -99,6 +103,7 @@ class CheckoutPlatformApi(
         }
     }
 
+    @SuppressLint("RestrictedApi")
```

Comment:
Interesting... I guess we should move `LogLevel` and `NONE` out of `Logger` on our side